### PR TITLE
fix for CharSerializer

### PR DESCRIPTION
**Pull request by alexpantyukhin**
Fix for test using Dictionary<byte, char> #https://github.com/akkadotnet/Hyperion/issues/2

### CODE REVIEW

File: Wire/ValueSerializers/CharSerializer.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ public CharSerializer() : base(Manifest, () => WriteValueImpl, () => ReadValueIm
         public static char ReadValueImpl(Stream stream, byte[] bytes)
         {
             stream.Read(bytes, 0, Size);
-            return (char) BitConverter.ToSingle(bytes, 0);
```
Comment (maxim-s) [inline Wire/ValueSerializers/CharSerializer.cs]: it's very strange https://msdn.microsoft.com/en-us/library/system.bitconverter.tosingle(v=vs.110).aspx

File: Wire/ValueSerializers/CharSerializer.cs [hunk 2]
```diff
@@ -16,7 +16,7 @@ public CharSerializer() : base(Manifest, () => WriteValueImpl, () => ReadValueIm
...
+            return BitConverter.ToChar(bytes, 0);
```
Comment (maxim-s) [inline Wire/ValueSerializers/CharSerializer.cs]: 👍

**Comment by Horusiath** [comment]
Good catch 👍  But you'll need to rebase with dev first.

**Comment by alexpantyukhin** [comment]
@Horusiath done.

**Comment by Aaronontheweb** [comment]
@Horusiath @alexpantyukhin any unit tests that cover this?

**Comment by alexpantyukhin** [comment]
@Aaronontheweb yes. https://github.com/akkadotnet/Hyperion/blob/dev/Hyperion.Tests/CollectionTests.cs#L67

**Comment by Aaronontheweb** [comment]
@alexpantyukhin oh great! Yep, could you remove the `Skip` on that?

**Comment by alexpantyukhin** [comment]
@Aaronontheweb done.

**Pull request merged**